alibi_detect/saving/validate.py

```diff
@@ -5,6 +5,41 @@
 from alibi_detect.version import __config_spec__, __version__
 
 
+ALLOWED_DETECTORS = [
```

Comment:
Nitpick - is `validate` the right place for this? I'd argue its more to do with typing than validating configs. We should at least add a comment saying what its for IMO.